docs/api-docs/cart-and-checkout/checkout-sdk.md

```diff
@@ -2,7 +2,7 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
+### On this page

```

Comment:
These sections need to be updated to match the changes below

Reply:
See PR 837.  This PR takes care of these changes.